src/form-field/__tests__/form-field-rendering.test.tsx

```diff
@@ -91,6 +91,20 @@ describe('FormField component', () => {
     expect(labelElement?.getElement().tagName).toBe('LABEL');
   });
 
+  test('errorIcon has an accessible text alternative', () => {
```

Comment:
Are these strings user-facing?

Reply:
These would appear in the console when tests are run during the development process.

Reply:
No it is not user-facing?